src/Engine/lib/api/routers/ProfileRouter.py

```
@@ -233,3 +233,56 @@ async def save_cookies(files: list[UploadFile] = File(...)) -> JSONResponse:
+    except ValueError as e:
+        logger.error(f"Error while adding account: {e}")
+        EngineContext.reset()
+        return JSONResponse(
+            status_code=400,
+            content={
+                "success": False,
+                "message": str(e)
+            }
+        )
+
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)